### PR TITLE
Logger configuration for file and console logging 

### PR DESCRIPTION
**Pull request by onurccn**
added logger configuration for logging package

Provided default usage

### CODE REVIEW

**Comment by onurccn** [comment]
[Backend Logger entegrasyonu](https://trello.com/c/XIGPM0IG/253-backend-logger-entegrasyonu)

**Review by aErgani — APPROVED** [review]
LGTM

File: backend/settings.py [hunk 1]
```diff
@@ -165,6 +159,44 @@
...
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "full": {
+            "format": "{levelname} - {asctime} - {pathname} - {funcName} - {process:d} - {thread:d} - {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "file": {
```
Comment (gurolcay) [inline backend/settings.py:172]: do we need this?

Comment (gurolcay) [reply backend/settings.py:172]: or why we need this?

**Review by gurolcay — APPROVED** [review]

**Pull request merged**